tests/Unit/Emogrifier/HtmlProcessor/CssInlinerTest.php

```diff
@@ -1243,7 +1268,11 @@ public function unneededCssThingsDataProvider()
             'style in "speech" media type rule' => ['@media speech {p {color: #000;}}', '#000'],
             'style in "tty" media type rule' => ['@media tty {p {color: #000;}}', '#000'],
             'style in "tv" media type rule' => ['@media tv {p {color: #000;}}', '#000'],
-        ];
+            'style in "only tv" media type rule' => ['@media only tv {p {color: #000;}}', '#000'],
+        ] + $this->getDatasetsWithVaryingWhitespace([
+            'style in "tv" media type rule' => [' @media tv { p { color : #000 ; } } ', '#000'],
+            'style in "only tv" media type rule' => [' @media only tv { p { color : #000 ; } } ', '#000'],
```

Comment:
I'd really prefer to have these data sets as string literals here instead of using getDatasetsWithVaryingWhitespace. Unit tests (and data provider data) should read like a story, and all that logic make it hard to read it like a story from top to bottom.